web3/contract/contract.py

```diff
@@ -168,29 +182,28 @@ def get_logs(
           same time as ``from_block`` or ``to_block``
         :yield: Tuple of :class:`AttributeDict` instances
         """
-        event_abi = self._get_event_abi()
-
+        abi = self._get_event_abi()
```

Comment:
nit: this seems like a less clarifying change. We could keep `event_abi` here instead of the more generic "abi".